gradle/javaProject.gradle

```diff
@@ -39,6 +39,8 @@ dependencies {
         implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.9.8'
         implementation 'com.h2database:h2:1.4.199'
         implementation 'org.hsqldb:hsqldb:2.2.7'
+        implementation 'ch.vorburger.mariaDB4j:mariaDB4j:2.4.0'
+        implementation 'mysql:mysql-connector-java:8.0.19'
```

Comment:
Why do we need these dependencies here?

Reply:
This is required for mysql module unit tests.